distil/primitives/mi_ranking.py

```diff
@@ -40,6 +42,16 @@ class Hyperparams(hyperparams.Hyperparams):
         semantic_types=['https://metadata.datadrivendiscovery.org/types/ControlParameter'],
         description="If True will return each columns rank in their respective metadata as the key 'rank'"
     )
+    sub_sample = hyperparams.Hyperparameter[bool](
+        default=False,
+        semantic_types=['https://metadata.datadrivendiscovery.org/types/ControlParameter'],
+        description="Whether or not to run MI ranking on a subset of the dataset"
+    )
+    sub_sample_size = hyperparams.Hyperparameter[typing.Optional[int]](
+        default=100,
```

Comment:
You had previously mentioned 1000 looking like it was a good setting for the default.  Did this change?

Reply:
Right, I'll put it back to 1000